migrate/playbooks/rds-timescale-pg-dump.md

```diff
@@ -0,0 +1,278 @@
+The procedures outlined in this, as well as subsequent sections, are based on the
+assumption that an Ubuntu Amazon Machine Image (AMI) was selected during the
+setup of the EC2 instance.
+</Highlight>
+
+To prepare your EC2 instance for migration:
+1. Navigate to your EC2 instance and click on the "Connect" button.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-pg_dump/prepare-ec2-instance.jpeg"
+ alt="Connect to EC2 instance"/>
+
+2. Select the "SSH client" tab.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-pg_dump/select-ssh-client.jpeg"
+ alt="Use SSH client"/>
+
+3. Connect to your EC2 instance using the "Key pair" downloaded in the previous step.
+```sh
+chmod 400 <key-pair>.pem
+ssh -i "<key-pair>.pem" ubuntu@<EC2 instance's Public IPv4>
+```
+
+4. Install PostgreSQL client.
+```sh
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+wget -qO- https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/trusted.gpg.d/pgdg.asc &>/dev/null
+sudo apt update
+sudo apt install postgresql-client-15 -y # "postgresql-client-16" if your source DB is using PG 16.
+psql --version && pg_dump --version
+```
+
+5. To allow an EC2 instance to connect to an RDS instance, you need to modify the
+security group associated with your RDS instance.
+
+5.a. Note the Private IPv4 address of your EC2 instance.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-pg_dump/private-ipv4-address.jpeg"
+ alt="Note private IPv4 address of your EC2 instance"/>
+
+5.b. Go to the security group associated with your RDS instance and select
+"Edit inbound rules".
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-pg_dump/modify-firewall-rules.jpeg"
+ alt="Edit inbound rules"/>
+
+5.c. Choose "Add rule". In the "Type" field, select "PostgreSQL". For "Source", select
+"Custom" and input the Private IPv4 address of your EC2 instance. Add a
+suitable description for this rule. Finally, click on "Save rules" to apply the changes.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-pg_dump/add-firewall-rule.jpeg"
+ alt="Add a new rule"/>
+
+6. Verify the connection to the RDS service from your EC2 instance. Please note,
+you will be prompted to enter the master password. This should be the same password
+that you used when creating your AWS RDS service.
+```sh
+# psql -h <rds_endpoint> -p <rds_port> -U postgres -c "select 1"
+psql -h aws-rds-migration-guide-db.csupoydrdg9f.us-east-1.rds.amazonaws.com -p 5432 -U postgres -c "select 1"
+Password for user postgres:
+ ?column?
+----------
+        1
+(1 row)
+```
+
+## Performing data migration
+
+<GettingHelp />
+
+<SourceTargetNote />
```

Comment:
These two notes one after the other doesn't look nice, makes it look cluttered.

<img width="654" alt="image" src="https://github.com/timescale/docs/assets/1117749/6e0c7171-c52b-439f-bdff-b0a7de1a9335">